src/connector/src/source/filesystem/s3/mod.rs

```diff
@@ -33,8 +34,10 @@ pub struct S3Properties {
     #[serde(rename = "match_pattern", default)]
     pub match_pattern: Option<String>,
     #[serde(rename = "s3.credentials.access", default)]
+    #[mark_redaction(rename = "s3.credentials.access")]
```

Comment:
should we also mark this as credentials? access key seems not that sensitive. cc @hzxa21